src/lib.rs

```diff
@@ -138,11 +139,15 @@ impl PrivateKeyRotationCheckResult {
         }
     }
 
-    //    pub fn group_rotation_needed() -> Option<Vec<GroupId>> {
-    //        unimplemented!()
-    //    }
+    pub fn group_rotation_needed(&self) -> Vec<GroupId> {
+        match &self.rotations_needed {
+            EitherOrBoth::Right(groups) | EitherOrBoth::Both(_, groups) => groups.to_vec(),
+            _ => vec![],
+        }
+    }
 
-    //    pub fn rotate_all(ironoxide: &IronOxide) -> (UserId, Vec<GroupId>) //TODO consider when group private key rotation is added
+    // TODO (reviewers): Is this something we want and I should implement as part of this ticket?
```

Comment:
Seems like a convenience we should write later. I'd write a ticket and leave it.

Reply:
I agree that delaying implementation here is fine. I think the most common use case for initializing with rotation is that you will want to just do any rotations that are needed, so it would simplify the caller's code significantly if we provide this.